ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/services/DefaultKsqlClient.java

```diff
@@ -122,7 +132,11 @@ public void makeAsyncLagReportRequest(
     final KsqlTarget target = sharedClient
         .target(serverEndPoint);
 
-    getTarget(target, authHeader).postAsyncLagReportingRequest(lagReportingMessage);
+    getTarget(target, authHeader).postAsyncLagReportingRequest(lagReportingMessage)
+        .exceptionally(t -> {
+          LOG.error("Unexpected exception in async request", t);
```

Comment:
Don't we want to debug log this as well?

Reply:
Looking at the code, it seems like all of the hosts start off in a non-alive state and if the hoststatus callback happens at the wrong time, I think lag reporting could hit the same issue as well.

Making it debug logging as well.